kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -548,7 +697,7 @@ public void shutdown() {
         this.interrupt();
         shutdownLatch.await();
       } catch (InterruptedException e) {
-        throw new Error("Interrupted when shutting down consumer worker thread.");
+        throw new Error("Interrupted when shutting down expiration thread.");
```

Comment:
nit: may as well be consistent and use `RuntimeException`? The type `Error` is usually reserved for abnormal situations such as out of memory.

Reply:
Absolutely. Good catch